### PR TITLE
make the flutter icon in the outline view more subdued

### PR DESCRIPTION
**Pull request by devoncarew**
- make the flutter icon in the outline view more subdued

This is similar to what the IntelliJ structure view does for Java elements.

screen shot 2018-02-22 at 12 52 08 pm

@scheglov

### CODE REVIEW

File: src/io/flutter/preview/PreviewView.java [hunk 1]
```diff
@@ -766,14 +766,17 @@ Icon getFlutterDecoratedIcon() {
...
     if (decorated == null) {
-      final Icon prefix = FlutterIcons.Flutter;
+      final Icon badgeIcon = FlutterIcons.Flutter_badge;
 
       decorated = new LayeredIcon(2);
-      decorated.setIcon(prefix, 0, 0, 0);
-      decorated.setIcon(icon, 1, prefix.getIconWidth(), 0);
+      decorated.setIcon(icon, 0, 0, 0);
+      decorated.setIcon(badgeIcon, 1,
+                        icon.getIconWidth() + (icon.getIconWidth() - badgeIcon.getIconWidth()) / 2,
```
Comment (scheglov) [inline src/io/flutter/preview/PreviewView.java]: What `(icon.getIconWidth() - badgeIcon.getIconWidth()) / 2` does?
It looks like we need a whitespace between these two items, but the way we compute is peculiar :-)

Comment (devoncarew) [reply src/io/flutter/preview/PreviewView.java]: It's a 8x8 icon centered in a 16x16 area (as if there were two 16x16 icons right next to each other).

Comment (scheglov) [reply src/io/flutter/preview/PreviewView.java]: Ah, makes sense.
Thanks.

**Comment by devoncarew** [comment]
some slight additional tweaks to the look:

screen shot 2018-02-22 at 8 48 18 pm

**Pull request merged**